tf2onnx/tfonnx.py

```diff
@@ -292,6 +292,20 @@ def placeholder_op(ctx, node, name, args):
     return None
 
 
+def placeholder_with_default_op(ctx, node, name, args):
+    default_value_node = ctx.get_node_by_name(node.input[0])
+    if default_value_node.is_const():
+        # Clone the tensor, set its name as current node's output, add as initializer
+        t = onnx_pb.TensorProto()
```

Comment:
refer to const_op?

def const_op(ctx, node, name, args):
    """Constants - make those initializers."""
    tensor = node.get_attr("value")
    ctx.add_initializer(tensor.t)

Reply:
talked offline with Guoliang.

two ops in original tf graph. one is placeholder, one is const. the placeholder is referencing the const as default value. So here need to parse tensor value from the const, and rename the tensor's name with placeholder's name. then the placeholder will become a input node, with an initializer containing the tensor.

for the const node in original graph, it will be handled as a normal constant, and will be deleted if no other node referenced it.